### PR TITLE
Fixed UG for overall descriptions and rate, changed aesthetics for html page

### LINKED ISSUE #215: [PE-D][Tester E] Minor grammar typo in rate command of User Guide
There seems to be a minor grammar issue at "If any of employee ID...", which makes reading it a bit confusing.

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: kangzongxian/ped#22

### PR DESCRIPTION
**Pull request by Kok-je**
Fixes #215 #157 #210 #200 for documentation.

Updated:
- Introduction Catchy Phrase
- Overall description
- Grammar typo in `rate` command
- Performance rating graph description
- rate once per day **note**
- Aesthetic changes to the HTML site for colors

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ title: User Guide
 
 Welcome to Coydir's User Guide! 
 
-**[Insert some catchy phrase here]**
+**Even Thanos had to read the User Guide before he learnt how to wield the power of the infinity stones...**
+
+**This guide will teach you the ropes so that you too can command the infinite power of Coydir the universe has never seen before!! Or simple feel free to look around! :p**
```
Comment (kevinchangjk) [inline docs/UserGuide.md:15]: "Or simple feel free to look around!"

**Review by kevinchangjk — CHANGES_REQUESTED** [review]
Might want to change the grammar a bit.

File: docs/UserGuide.md (hunk 1 above)
Comment (chao890) [reply docs/UserGuide.md:15]: Simply change here

**Review by chao890 — APPROVED** [review]

**Review by chao890 — APPROVED** [review]
LGTM

**Review by kevinchangjk — APPROVED** [review]
LGTM.

**Pull request merged**